drivers/download_stage1_psuedo.R

```diff
@@ -15,27 +17,40 @@ dates_pseudo <- seq(as.Date("2020-09-24"), Sys.Date(), by=1)
 
 message("GEFS v12 stage1-stats")
 bench::bench_time({ # thelio
-  s3 <- gefs_s3_dir("stage1-stats")
+  s3 <- gefs4cast::gefs_s3_dir(product = "stage1-stats",
+                               path = "", # should this path be more specific? the noaa bucket in the config is "drivers/noaa/gefs-v12-reprocess/"
```

Comment:
@rqthomas @cboettig  should we include the project_id of the challenge in `path`? as it is right now, this will return: 
```
s3://bio230014-bucket01/challenges/drivers/gefs-v12/stage1-stats/
```

`path = config$project_id` would return the following path: 
```
s3://bio230014-bucket01/challenges/drivers/usgsrc4cast/gefs-v12/stage1-stats/
```

Reply:
Yes - see above